postcss.config.js

```diff
@@ -2,5 +2,15 @@ module.exports = {
   plugins: {
     tailwindcss: {},
     autoprefixer: {},
+    'postcss-preset-mantine': {},
+    'postcss-simple-vars': {
+      variables: {
+        'mantine-breakpoint-xs': '36em',
+        'mantine-breakpoint-sm': '48em',
```

Comment:
interesting to go for em

Reply:
this is their out of the box recommendation but we'll discuss the values later on and adapt them if needed